internal/policy/operator/deployable_by_olm.go

```diff
@@ -267,7 +273,7 @@ func (p *DeployableByOlmCheck) setUp(ctx context.Context, operatorData *operator
 			return err
 		}
 	} else {
-		log.L().Debug("No docker config file is found to access the index image in private registries. Proceeding...")
+		logger.V(1).Info("no docker config file is found to access the index image in private registries")
```

Comment:
+1 to removing `Proceeding...`, but should we instead consider saying something like `using anonymous auth`?